docs/sources/mimir/send/native-histograms/_index.md

```diff
@@ -0,0 +1,268 @@
+
+The value of `NativeHistogramMaxBucketNumber`/`nativeMaxNumberOfBuckets` limits the number of buckets produced by the observations. This can be especially useful if the receiver side is limiting the number of buckets that can be sent. For more information about the bucket limit see [Limiting the number of buckets](#limiting-the-number-of-buckets).
+
+The duration in `NativeHistogramMinResetDuration`/`nativeResetDuration` will prohibit automatic counter resets inside that period. Counter resets are related to the bucket limit, for more information see [Limiting the number of buckets](#limiting-the-number-of-buckets).
+
+### Scrape and send native histograms with Prometheus
+
+Use the latest version of Prometheus or at least version 2.47.
+
+1. To enable scraping native histograms from the application, you need to enable native histograms feature via a feature flag on the command line:
+
+   ```bash
+   prometheus --enabled-feature native-histograms
+   ```
+
+1. The above flag will make Prometheus detect and scrape native histograms, but ignores classic histogram version of metrics. In case of a migration when a metric is both a classic and native histogram at the same time you need to set `scrape_classic_histograms` to `true` in your scrape jobs, for example:
+
+   ```yaml
+   scrape_configs:
+     - job_name: myapp
+       scrape_classic_histograms: true
+   ```
+
+   in your scrape jobs, to get both histogram version.
+
+   {{% admonition type="note" %}}
+   Native histograms don't have a textual presentation at the moment on the application's `/metrics` endpoint, thus Prometheus negotiates a Protobuf protocol transfer in this case.
+   {{% /admonition %}}
+
+1. To be able to send native histograms to a Prometheus remote write compatible receiver, for example Grafana Cloud Metrics, Mimir, etc, set `send_native_histograms` to `true` in the remote write configuration, for example:
+
+   ```yaml
+   remote_write:
+     - url: http://.../api/prom/push
+       send_native_histograms: true
+   ```
+
+### Scrape and send native histograms with Grafana Agent
+
+Use the latest version of the Grafana Agent in [Flow mode](https://grafana.com/docs/agent/latest/flow/) or at least version 0.37.
+
+1. To enable scraping native histograms you need to enable the argument `enable_protobuf_negotiation` in the `prometheus.scrape` component:
+
+   ```
+   prometheus.scrape "myapp" {
+     enable_protobuf_negotiation = true
+   }
+   ```
+
+1. The above flag will make Grafana Agent detect and scrape native histograms, but ignores classic histogram version of metrics. In case of a migration when a metric is both a classic and native histogram at the same time you need to set `scrape_classic_histogram` to `true` in your scrape jobs, for example:
```

Comment:
```suggestion
1. The above flag will make Grafana Agent detect and scrape native histograms, but ignores classic histogram version of metrics. In case of a migration when a metric is both a classic and native histogram at the same time you need to set `scrape_classic_histograms` to `true` in your scrape jobs, for example:
```